stories/navbar.stories.js

```diff
@@ -0,0 +1,180 @@
+import { hbs } from 'ember-cli-htmlbars';
+import { withKnobs, boolean } from '@storybook/addon-knobs';
+import { argument } from './knob-categories';
+
+export default {
+  title: 'DenaliNavbar',
+  component: 'DenaliNavbar',
+  decorators: [withKnobs],
+};
+
+export const Default = () => ({
+  template: hbs`
+  <DenaliNavbar
+        @isResponsive={{isResponsive}}
+        @isMenuActive={{isMenuActive}}
+        as |Nav| >
+    <Nav.Left as |Section|>
+      <Section.Logo src="https://denali-design.github.io/denali-css/denali-logo.svg"/>
```

Comment:
Can we refer to a local copy? If you rebase, you can copy this file into `.storybook/static` and make this:
```suggestion
      <Section.Logo src="denali-logo.svg"/>
```

Reply:
Hmmm I can't seem to get this to work.  Based on the logs it doesn't seem like it copies things over from here...

info => Loading static files from: /Users/cwilliamson/Documents/denali-ember/dist .
info => Loading presets
info => Loading presets
info => Loading config/preview file in "./.storybook".
info => Loading config/preview file in "./.storybook".
info => Adding stories defined in ".storybook/main.js".
info => Using default Webpack setup

Reply:
Did you rebase?